src/framework/audio/internal/synthesizers/fluidsynth/fluidsequencer.cpp

```diff
@@ -274,14 +274,6 @@ int FluidSequencer::expressionLevel(const mpe::dynamic_level_t dynamicLevel) con
 
     float stepCount = ((dynamicLevel - MIN_SUPPORTED_LEVEL) / static_cast<float>(mpe::DYNAMIC_LEVEL_STEP));
 
-    if (dynamicLevel == mpe::dynamicLevelFromType(DynamicType::Natural)) {
-        stepCount -= 0.5;
-    }
-
-    if (dynamicLevel > mpe::dynamicLevelFromType(DynamicType::Natural)) {
-        stepCount -= 1;
-    }
-
```

Comment:
This appeared to be a hack on the fluid synth to ignore this "natural" dynamics layer; it is not needed anymore.